.tugboat/config.yml

```diff
@@ -1,26 +1,21 @@
 services:
-  # What to call the service hosting the site. Because there is only
-  # one service, it is automatically set as the default service, which
-  # does a few things
-  #   1. Clones the git repository into the service container
-  #   2. Exposes port 80 to the Tugboat HTTP proxy
-  #   3. Routes requests to the preview URL to this service
   apache:
-
-    # Use the available version of Apache by not specifying a version
     image: tugboatqa/httpd
-
-    # A set of commands to run while building this service
     commands:
-
-      # Commands that set up the basic preview infrastructure
+      # Used in the base image as setup.
       init:
         - |
           curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.0/install.sh | bash
           NVM_DIR="$HOME/.nvm"
           [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
           nvm install 10.15.1
```

Comment:
Do we need to update this to 11.14.0 also?

Reply:
That would probably be good. Thanks for catching that.